helm/aws-efs-csi-driver/templates/node.yaml

```diff
@@ -3,7 +3,6 @@ kind: DaemonSet
 apiVersion: apps/v1
 metadata:
   name: efs-csi-node
-  namespace: kube-system
```

Comment:
name as above, we might want to set the namespace

Reply:
fixed